### PR TITLE
remove additional chapter label section headers answerkey

### PR DESCRIPTION
**Pull request by ajluk**
Issue: #2061
This PR will resolve also issue #2064

### CODE REVIEW

**Comment by ajluk** [comment]
Now it looks like this (University Physics vol1 PDF file): 
image

File: recipes/mixins/_compose.scss [hunk 1]
```diff
@@ -684,19 +684,20 @@
...
             }
-            &::after {
-              content: nodes($sectionHeaderNode);
-              move-to: moduleTitleName;
-            }
+            //Left commented just in case the section headers will be needed in the future
+            // &::after {
+            //   content: nodes($sectionHeaderNode);
+            //   move-to: moduleTitleName;
+            // }
```
Comment (wilkus27) [inline recipes/mixins/_compose.scss]: Why is this removed? It adds section titles.

Comment (ajluk) [reply recipes/mixins/_compose.scss]: In the old pdf form opentstax.org  the sections aren't present. I have asked Larissa about it and she said it is ok.

Comment (wilkus27) [reply recipes/mixins/_compose.scss]: Ok, but this change removed also section titles from Precalculus where we want them.

**Comment by wilkus27** [comment]
@ajluk I dive into it more carefully. Here is how mixin `compose_createEOBSolutions` works.

In config `$Config_BookCompositePages` we can choose if solutions are clustered by chapter or section. In the first case (e.g applied to Chemistry) it looks like that:

image

All solutions from the whole chapter are placed next to each other.

In the second case we have two results. First:

image

There are added titles of the EOC sections. And regular section titles in answers from section notes.  And they are added by code which you removed.

I checked why they are needed in Precalculus. So in this book numbering of the notes resets by module:

image

That's why we need those headers to know from which module are they. 
So definitelly this condition from the mixin is wrong:

image

(file: `_compose.scss`)

I think we can use this information:

image

(file: config.scss for Precalculus)

I mean, that we can create condition that those titles will be added in case when key `resetAt` in `$Config_Notes` is equal to `moduleReset`.

Please let me know if I should explain something more precisely.

**Comment by ajluk** [comment]
@wilkus27 I have added the suggested changes and the code is ready for review. Thanks for the idea!

File: styles/designs/carnival/parts/_answerkey-components.scss [hunk 2]
```diff
@@ -70,6 +70,19 @@ $AnswerKeyExerciseTitleTextComponent: (
...
+  _properties: (
+    color: enum('ValueSet:::REQUIRED'),
+    font-family: enum('ValueSet:::REQUIRED'),
+    font-size: font-scale(3),
+    line-height: line-height(3),
+    font-weight: bold,
+  )
+);
+
+
```
Comment (wilkus27) [inline styles/designs/carnival/parts/_answerkey-components.scss]: I think we don't need this component now. 

image

This element has now `os-solution-area` class like other elements.

Comment (ajluk) [reply styles/designs/carnival/parts/_answerkey-components.scss]: Thanks @wilkus27! I have missed that! The component has been removed.

**Comment by brittweinstein** [comment]
@ajluk we can merge w/o BrittJ's review since Alan has approved :)
